### PR TITLE
Use PopularTable model when return user resource

### PR DESCRIPTION
**Pull request by feng-tao**
### Summary of Changes

When building Amundsen people, we found that it is not necessary to use Table model when return table object to FE. The table model contains all the metadata related table detail page which the payload size is huge. Switch to use popularTable model.

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`
- [x] I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by feng-tao** [comment]
cc @danwom @jinhyukchang

File: metadata_service/api/user.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from flask_restful import Resource, fields, marshal
 
-from metadata_service.api.table import table_detail_fields
+from metadata_service.api.popular_tables import popular_table_fields
```
Comment (danwom) [inline metadata_service/api/user.py]: Can you rename `popular_table_fields` to something generic like `table_summary_fields` and move it to a common file for tables in general?

Comment (danwom) [reply metadata_service/api/user.py]: Also I think `popular_table_fields` doesn't include `last_updated_epoch`. I think the data model used by search service is more complete.

Comment (feng-tao) [reply metadata_service/api/user.py]: this has been used by atlas proxy as well which we could do but will be in a different pr.

**Pull request merged**